sbin/functions.sh

```diff
@@ -966,6 +966,189 @@ clean_up()
 	[ -n "${mnt2}" ] && umount ${mnt2}
 }
 
+interactive_config()
+{
+	local device="$1"
+	local localconf_dir="$2"
+	local tmpfile=$(mktemp)
+
+	if [ -z ${device} ] || [ -z ${localconf_dir} ]
```

Comment:
We should be checking the input device before here .. I thought we were. But either way, we should clearly state which parameters were missing in the error message.